packages/maskbook/src/database/Persona/Persona.db.ts

```diff
@@ -64,8 +71,13 @@ const db = createDBAccessWithAsyncUpgrade<PersonaDB, Knowledge>(
                         }
                     }
                 }
+                async function v2_v3() {
+                    db.createObjectStore('relations', { keyPath: ['linked', 'profile'] })
+                    transaction.objectStore('relations').createIndex('linked', 'linked', { unique: false })
```

Comment:
Why don't you add `favor` into the index? Isn't it a requirement to order all favored contacts at the most front?

Reply:
I added a compound index to do this feature.